modules/virt-hot-plugging-cpu.adoc

```diff
@@ -19,3 +19,12 @@ You can increase or decrease the number of CPU sockets allocated to a virtual ma
 +
 If the VM is migratable, a live migration is triggered. If not, or if the changes cannot be live-updated, a `RestartRequired` condition is added to the VM.
 
+[NOTE]
+====
+If a VM has the `spec.template.spec.domain.devices.networkInterfaceMultiQueue` field enabled and CPUs are hot-plugged, the following behavior occurs:
```

Comment:
```suggestion
If a VM has the `spec.template.spec.domain.devices.networkInterfaceMultiQueue` field enabled and CPUs are hot plugged, the following behavior occurs:
```

I'm basing this on doc repo searches for similar examples. The hyphen form would need to a closing noun.